qutebrowser/browser/webelem.py

```diff
@@ -50,6 +50,7 @@
     Group.url: '[src], [href]',
     Group.prevnext: 'a, area, button, link, [role=button]',
     Group.focus: '*:focus',
+    Group.inputs: 'input[type=text], input[type=password], textarea',
```

Comment:
I used the example in #219. 
```input:not([type=hidden])``` does look better but on duckduckgo's homepage, for example, it also selects the 'clear' and 'submit' buttons.